os/arch/xtensa/src/esp32/esp32_i2c.c

```diff
@@ -362,7 +362,7 @@ static int i2c_esp32_isr(int irq, FAR void *context, FAR void *arg)
 					break;
 				}
 
-				/* not need to give seamphore, do work in isr itself*/
+				/* not need to give semaphore, do work in isr itself*/
```

Comment:
Please add space before closing the comment `*/`

Reply:
You're right. I'll fix it